src/rt-app_args.c

```diff
@@ -42,6 +43,7 @@ char help_full[] = \
 "through the standard input.\n\n"
 "Miscellaneous:\n"
 "  -v, --version             display version information and exit\n"
+"  -l, --log                 set log level (0...100 recommended, 50 default)\n"
```

Comment:
Why recommended?

Reply:
because other values don't make sense. actually, the lowest value that makes sense is 10 (ERROR and CRITICAL), the highest is 100.

i was trying to keep within the string length, but could elaborate, if you like...

Reply:
Ah OK sorry.. I misinterpreted as if 100 was the recommended value.
Maybe we could just remove "recommended" and check that user don't pick something outside the range?